From 6423c413619bf03795716c4db46d37e447b28e29 Mon Sep 17 00:00:00 2001
From: Zain Wang <wzz@rock-chips.com>
Date: Wed, 4 May 2016 09:04:39 +0800
Subject: [PATCH] regulator: mp8865: update mp8865 enable_time

enable_time is controlled by capacitance connected to Pin SS,
it described more detailedly in mp8865 datasheet Page 15.
Capacitance is 12nF now.

Change-Id: Ib604a4e109db7ab125104e5cf3067864fefb6fe0
Signed-off-by: Zain Wang <wzz@rock-chips.com>
---
 drivers/regulator/mp8865-regulator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/regulator/mp8865-regulator.c b/drivers/regulator/mp8865-regulator.c
index a8d769cff123..ec8b63baef73 100644
--- a/drivers/regulator/mp8865-regulator.c
+++ b/drivers/regulator/mp8865-regulator.c
@@ -21,7 +21,7 @@
 #define MP8865_MAX_VSEL			0x7f
 #define VOL_MIN_IDX			0x00
 #define VOL_MAX_IDX			0x7f
-#define MP8865_ENABLE_TIME		100
+#define MP8865_ENABLE_TIME		150
 
 /* Register definitions */
 #define MP8865_VOUT_REG			0
-- 
2.35.3

